central/declarativeconfig/health/datastore_impl.go

```diff
@@ -0,0 +1,75 @@
+package health
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/central/declarativeconfig/health/store"
+	"github.com/stackrox/rox/central/role/resources"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/errox"
+	"github.com/stackrox/rox/pkg/postgres/pgutils"
+	"github.com/stackrox/rox/pkg/sac"
+)
+
+var (
+	integrationSAC = sac.ForResource(resources.Integration)
+)
+
+type datastoreImpl struct {
+	store store.Store
+}
+
+func (ds *datastoreImpl) GetDeclarativeConfigs(ctx context.Context) ([]*storage.DeclarativeConfigHealth, error) {
+	if ok, err := integrationSAC.ReadAllowed(ctx); err != nil {
```

Comment:
(completely unrelated with the PR):
we should probably think about creating another util function in SAC code similar to `VerifyAuthZOK` which doesn't return an error in case `ok == false`, and document it properly so that each API behaves the same (on GET return no `401`, on other methods return `401` in case no permissions).